.github/dependabot.yaml

```diff
@@ -0,0 +1,16 @@
+version: 2
```

Comment:
Add a copyright header, similar to [this](https://github.com/hashicorp/go-eventlogger/pull/135/files).